inference-engine/src/mkldnn_plugin/mkldnn_exec_network.h

```diff
@@ -43,6 +43,10 @@ class MKLDNNExecNetwork: public InferenceEngine::ExecutableNetworkThreadSafeDefa
     INFERENCE_ENGINE_DEPRECATED("Use InferRequest::QueryState instead")
     std::vector<InferenceEngine::IVariableStateInternal::Ptr> QueryState() override;
 
+    void Export(const std::string &modelFileName) override;
```

Comment:
it already has default implementation, can we remove this override?

Reply:
ok